### PR TITLE
[Tests-Only] changed storage providers to ocis for ocis service

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- storages drivers are changed to `ocis` for web-tests

## Related Issue

- Fixes https://github.com/owncloud/web/issues/5038

## How Has This Been Tested?

- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15231/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15234/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15237/

<p>










</p></details>

File: tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md [hunk 1]
```diff
@@ -439,3 +436,240 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
+-   [webUIDeleteFilesFolders/deleteFilesFolders.feature:155](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L155)
+-   [webUIDeleteFilesFolders/deleteFilesFolders.feature:176](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L176)
+-   [webUIDeleteFilesFolders/deleteFilesFolders.feature:245](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L245)
+-   [webUIDeleteFilesFolders/deleteFilesFolders.feature:268](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L268)
+
+-   [webUIPreview/imageMediaViewer.feature:121](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L121)
+
+- [webUIPreview/imageMediaViewer.feature:121](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIPreview/imageMediaViewer.feature#L121)
+
```
Comment (jasson99) [inline tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md:449]: Aren't these same?

**Comment by jasson99** [comment]
I would prefer to add the expected failures list in sequential order of the tests. That would make future manual adjustments less complex I think.

File: tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md [hunk 2]
```diff
@@ -51,7 +51,6 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIFilesDetails/fileDetails.feature:51](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesDetails/fileDetails.feature#L51)
 
 ### [Sharing seems to work but does not work](https://github.com/owncloud/ocis/issues/1303)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md:53]: This file should be renamed `expected-failures-with-ocis-server-ocis-storage.md`
And the same change somewhere in `.drone.star`

File: tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md [hunk 3]
```diff
@@ -439,3 +436,240 @@ Other free text and markdown formatting can be used elsewhere in the document if
 
 -   [webUISharingAcceptShares/acceptShares.feature:100](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L100)
 -   [webUISharingAcceptShares/acceptShares.feature:116](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L116)
+
+-   [webUIDeleteFilesFolders/deleteFilesFolders.feature:304](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L304)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md:440]: why is there such a long list of extra test scenarios that fail?

I hope that the `ocis` storage driver is better than the `owncloud` storage driver!

Maybe there are other changes needed to env variables etc in the pipelines?

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md [hunk 4]
```diff
@@ -440,7 +437,245 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
+-   [webUIMoveFilesFolders/moveFolders.feature:66](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIMoveFilesFolders/moveFolders.feature#L66)
+-   [webUIMoveFilesFolders/moveFolders.feature:12](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIMoveFilesFolders/moveFolders.feature#L12)
+-   [webUIMoveFilesFolders/moveFolders.feature:68](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIMoveFilesFolders/moveFolders.feature#L68)
+-   [webUIMoveFilesFolders/moveFolders.feature:69](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIMoveFilesFolders/moveFolders.feature#L69)
+-   [webUIMoveFilesFolders/moveFolders.feature:81](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIMoveFilesFolders/moveFolders.feature#L81)
+
+-   [webUICreateFilesFolders/createFolderEdgeCases.feature:44](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUICreateFilesFolders/createFolderEdgeCases.feature#L44)
+>>>>>>> a5b523e8 (updated expected failures for new test configuration)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:681]: This is a leftover from resolving conflicts.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15273/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15273/

<p>










</p></details>

File: tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md]: fixed

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15275/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15283/

<p>










</p></details>

File: tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md (hunk 3 above)
Comment (kiranparajuli589) [reply tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md]: actually `ocis` commit id was was old.

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by SwikritiT — COMMENTED** [review]
LGTM

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md [hunk 5]
```diff
@@ -443,4 +413,16 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
+
+-   [webUISharingInternalGroups/shareWithGroups.feature:245](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature#L245)
+-   [webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:80](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature#L80)
+-   [webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:121](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsersSharingIndicator/shareWithUsers.feature#L121)
+
+-   [webUITrashbinDelete/trashbinDelete.feature:106](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L106)
+-   [webUITrashbinDelete/trashbinDelete.feature:86](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L86)
+-   [webUITrashbinDelete/trashbinDelete.feature:50](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L50)
```
Comment (individual-it) [inline tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:428]: do all those tests fail because of https://github.com/owncloud/web/issues/1158 ?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15330/

<p>


### webUIMarkdownEditor-markdownFile-feature-71.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15330/

<p>


### webUIMoveFilesFolders-moveFiles-feature-105.png
  


</p></details>

**Comment by phil-davis** [comment]
@kiranparajuli589 is this still blocked?

**Comment by kiranparajuli589** [comment]
> @kiranparajuli589 is this still blocked?

it's not blocked now, I've linked new failures with the issues.
please review.

File: tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature [hunk 6]
```diff
@@ -75,6 +75,10 @@ Feature: deleting files and folders
 
   @skipOnOC10 @issue-4582
   Scenario: Delete all files at once
+    Given user "Alice" has uploaded file "data.zip" to "data.zip"
+    And user "Alice" has created file "lorem.txt"
+    And user "Alice" has created folder "simple-folder"
+    And the user has browsed to the files page
     When the user marks all files for batch action using the webUI
     And the user batch deletes the marked files using the webUI
     And there should be no resources listed on the webUI
```
Comment (phil-davis) [inline tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature:84]:
```suggestion
    Then there should be no resources listed on the webUI
```
This is to do in another PR - I guess there are some missing `Then` keywords, and I happened to notice this one.

**Review by phil-davis — APPROVED** [review]
LGTM

**Review by individual-it — APPROVED** [review]

**Comment by phil-davis** [comment]
:rocket:
